docs/UserGuide.md

```diff
@@ -243,9 +326,31 @@ _Details coming soon ..._
 
 --------------------------------------------------------------------------------------------------------------------
 
+## Calendar
+
+![Calendar](images/calendarView.png)
+
+The calendar UI allows you to visualize your appointments throughout the day.
+
+The **right side of the calendar** shows a timeline view of today's appointments. Each appointment is represented
+by a blue box with the person's name. For appointments lasting 30 minutes or more, the start time is also displayed.
+The height of the box represents the duration of the appointment.
+
+The **left side of the calendar** shows a monthly calendar view along with an agenda view that lists appointments 
+for the next 30 days, including today (e.g., November 7 to December 7). The agenda view provides detailed information
+for each appointment, including the person's name and the start and end times.
+
+You can also navigate the calendar using these keyboard shortcuts:
+
+* `Ctrl + P`: Go to the previous day
```

Comment:
Should we specify command + P etc. for Mac users as well?

Reply:
The `Ctrl` key is analogous to macOS's `control` key, so `Command + P` wouldn't work.